### ISSUE TITLE
Cannot add a custom header if not exists on openapi

### ISSUE THREAD

**Issue by nadir-albajari-hs** [apideck-libraries/portman#157]
Hi, I'm trying to add a custom header, but it seems that if it's not defined it will not add it 
please see comments on code...

```javascript
import { PostmanMappedOperation } from '../../postman'
import { OverwriteRequestHeadersConfig } from '../../types'

/**
 * Overwrite Postman request headers with values defined by the portman testsuite
 * @param overwriteValues
 * @param pmOperation
 */
export const overwriteRequestHeaders = (
  overwriteValues: OverwriteRequestHeadersConfig[],
  pmOperation: PostmanMappedOperation
): PostmanMappedOperation => {

  if (!(overwriteValues instanceof Array)) return pmOperation

/* 

if we ran on pmHeaders it's impossible to add a new header 

*/
  if (!pmOperation.item?.request?.headers) return pmOperation

  pmOperation.item.request.headers.each(pmHeader => {

    overwriteValues.forEach(overwriteValue => {

      if (!(overwriteValue.key && pmHeader.key && overwriteValue.key === pmHeader.key)) {
        return
      }

      if (overwriteValue?.value && pmHeader?.value) {
        const originalValue = pmHeader.value
        let newValue = overwriteValue.value

        if (overwriteValue.overwrite === false) {
          newValue = originalValue + newValue
        }
        pmHeader.value = newValue
      }


      if (overwriteValue.disable === true) {
        pmHeader.disabled = true
      }


      pmOperation.item.request.upsertHeader(pmHeader)
    })
  })


  overwriteValues
    .filter(({ remove }) => remove)
    .map(headerToRemove => {
      pmOperation.item.request.headers.remove(header => {
        return header.key === headerToRemove.key
      }, null)
    })
   /* 

  Maybe we can add a new option to add if not exists

  */
  overwriteValues
    .filter(({ add }) => add)
    .map(headerToAdd => {
      pmOperation.item.request.headers.add({key:overwriteValue.key, value:overwriteValue.value})
  })

  return pmOperation
}

```

**Comment by thim81** [comment]
@nadir-albajari-hs 

We are making progress on the development of the feature.
There is one thing still off, which needs to be fixed.

You can follow the progress of the PR #160

**Comment by thim81** [comment]
@nadir-albajari-hs Would you mind to review If the "insert" option is clear, as documented in the readme and the example: https://github.com/apideck-libraries/portman/tree/overwrite-headers-insert/examples/testsuite-overwrites#overwrite-options

**Comment by thim81** [comment]
@nadir-albajari-hs Closing this request, since the new feature is included in the 1.10.0 release
